.github/workflows/publish-example-images.yaml

```
@@ -42,12 +41,27 @@ jobs:
             dockerfile: examples/pytorch/cpu-demo/Dockerfile
             context: examples/pytorch/cpu-demo
           - component-name: xgboost-dist-iris
+            platforms: linux/amd64
```

```suggestion
            platforms: linux/amd64,linux/arm64
```

We could specify multiple platforms like this here.
So, could you apply the same approach to other places?